ports/yasm-tool/vcpkg.json

```
@@ -1,7 +1,7 @@
 {
   "name": "yasm-tool",
-  "version-string": "2020-10-12",
-  "description": "A modular assembler. This port is intended to build other ports and should not be used directly.",
+  "version": "1.3.0",
+  "description": "A modular assembler.",
   "homepage": "http://yasm.tortall.net/",
-  "supports": "windows & x86"
+  "supports": "windows & !uwp & (x86 | x64)"
```

```suggestion
  "supports": "!uwp"
```
This port supports non-windows platforms via `vcpkg_find_acquire_program()`.